manifests/b/buchen/portfolio/0.74.2/buchen.portfolio.installer.yaml

```diff
@@ -1,15 +1,14 @@
-# Created using wingetcreate 1.9.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+# Created with YamlCreate.ps1 v2.5.0 $debug=AUSU.CRLF.7-5-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 
 PackageIdentifier: buchen.portfolio
 PackageVersion: 0.74.2
-Scope: machine
```

Comment:
why is scope: machine being removed here?

Reply:
Reverted